CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins
 ### Added
 - Updated asset build targets to support centos6
 
+## [4.0.2] - 2019-11-11
```

Comment:
Per our [guidelines](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#the-version-diffs-for-maintainers) versioning is done by the maintainers to avoid additional conflicts when we accept multiple pull requests into the same release.